rados/watcher_test.go

```diff
@@ -0,0 +1,223 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package rados
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func (suite *RadosTestSuite) TestWatcher() {
+	suite.SetupConnection()
+	oid := suite.GenObjectName()
+	err := suite.ioctx.Create(oid, CreateExclusive)
```

Comment:
Can we add a defer here (under the NoError) to delete the object we created. I'd also accept deleting it at the bottom of the function, after the subtests, but I think a defer is more readable and keeps the cleanup logic "near" the creation. Either way, we should delete the objects the tests create. :-)